cliquet_docs/contributing.rst

```diff
@@ -112,3 +112,35 @@ on ``#storage`` on ``irc.mozilla.org``.
 * You can direct your IRC client to the channel using `this IRC link
   <irc://irc.mozilla.org/storage>`_ or you can manually join the #storage IRC
   channel on the mozilla IRC network.
+
+
+How to release
+==============
+
+In order to prepare a new release, we are following the following steps:
+
+1. Update the CHANGELOG with the changes since the last release;
+2. Update the version in setup.py;
```

Comment:
I like it to be clear and it prevent us for doing mistakes (forgetting the Back to development commit for instance)